src/emulator/modules/ScePower/src/ScePower.cpp

```diff
@@ -140,31 +140,49 @@ EXPORT(int, scePowerRequestSuspend) {
 }
 
 EXPORT(int, scePowerSetArmClockFrequency, int freq) {
+    if (freq < 0){
+        return SCE_POWER_ERROR_INVALID_VALUE;
+    }
     return 0;
 }
 
-EXPORT(int, scePowerSetBusClockFrequency) {
-    return unimplemented("scePowerSetBusClockFrequency");
+EXPORT(int, scePowerSetBusClockFrequency, int freq) {
+    if (freq < 0){
+        return SCE_POWER_ERROR_INVALID_VALUE;
+    }
+    return 0;
 }
 
-EXPORT(int, scePowerSetConfigurationMode) {
-    return unimplemented("scePowerSetConfigurationMode");
+EXPORT(int, scePowerSetConfigurationMode, int mode) {
+    if (mode != 0x80 && mode != 0x800 && mode != 0x10880){
```

Comment:
Do we have named constants for these values?

Reply:
vita-headers doesn't have them. I grabbed those values with IDA Pro.